caddyhttp/push/handler.go

```diff
@@ -58,26 +58,41 @@ outer:
 }
 
 func (h Middleware) servePreloadLinks(pusher http.Pusher, headers http.Header, links []string) {
```

Comment:
Can you add a godoc-style comment for this function that describes the syntax of the Link headers that it parses and applies?